code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -367,6 +367,11 @@
 	description = "A sleek black turtleneck paired with some khakis (WARNING DOES NOT HAVE SUIT SENSORS)"
 	path = /obj/item/clothing/under/syndicate/tacticool
 
+/datum/gear/uniform/suit/greyman
+	display_name = "Greyman Henley"
```

Comment:
```suggestion
	display_name = "\improper Greyman Henley"
```

Reply:
what does this do?

Reply:
This makes it match the other items in game since we don't have any capitalization for most (if any) items.

changes it to: `greyman henley` in game within the tooltip.

Reply:
> what does this do?

\improper makes the game treat it as an improper noun.

That way when you examine it, the narrator says "That's **a** Greyman Hanley" instead of "That's Greyman Hanley"

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/31545294/ce705d35-2adc-4246-abb0-adac77f95fa9)
i dont exactly understand why it needs to be there while its not there in tacticool

Reply:
>i dont exactly understand why it needs to be there while its not there in tacticool

If you look at the actual item rather than the outfit datum, you'll see "tacticool turtleneck" is lowercase, so does not require \improper.

Reply:
This is the outfit datum, not the clothing typepath.
These can be capitalized.

Reply:
🐦 👍